json/src/lib.rs

```diff
@@ -1167,6 +1167,41 @@ pub struct FinalizePsbtResult {
     pub complete: bool,
 }
 
+/// Models the result of "getchaintips"
+pub type GetChainTipsResult = Vec<GetChainTipsResultTip>;
```

Comment:
Hmm, I'd just have had the method return a Vec without the typedef, but sure, doesn't hurt other than confusing documentation.